SAMtags.tex

```diff
@@ -500,13 +508,14 @@ \subsubsection*{July 2010}
 \gap
 Tags FI, FS, OC, OP, OQ, and TC added.
 
-Tags GC, GQ, and GS listed as reserved for backwards compatibility.
+Tags GC:Z, GQ:Z, and GS:Z, briefly proposed for representing repeatedly-sequenced reads, noted as reserved for backwards compatibility.
+Existing tags MF:i (MAQ pair flag), SQ:H (suboptimal bases), and S2:H (mate's suboptimal bases) removed and noted as reserved for backwards compatibility.
 
-Existing CC, CP, and MF tags removed and noted as reserved for backwards compatibility.
+CC and CP tags temporarily removed.
```

Comment:
what does "temporarily" removed mean?

Reply:
It's clear in the context of the March 2011 entry.

Reply:
It confused me initially too, but maybe just add "(until March 2011)" as a hint.   It's all ancient history though and I doubt most people will be reading back that far.